packages/backend/src/server/api/endpoints/admin/show-user.ts

```diff
@@ -23,9 +23,9 @@ export const paramDef = {
 
 // eslint-disable-next-line import/no-default-export
 export default define(meta, paramDef, async (ps, me) => {
-	const user = await Users.findOneBy({ id: ps.userId });
+	const [user, profile] = await Promise.all([Users.findOneBy({ id: ps.userId }), UserProfiles.findOneBy({ userId: ps.userId })]);
```

Comment:
I think spreading this over multiple lines makes this easier to read.
```suggestion
	const [user, profile] = await Promise.all([
	 Users.findOneBy({ id: ps.userId }),
	 UserProfiles.findOneBy({ userId: ps.userId })
	]);
```